### DISCUSSION TITLE
Migrating from deprecated signalfx gateway using prometheus remote write

### DISCUSSION THREAD

**Discussion by timmow** [signalfx/splunk-otel-collector#2311] [General]
We are running the [signalfx gateway](https://github.com/signalfx/gateway) to receive from prometheus via remote write and send to signalfx - example config

```
{
  "LogDir": "-",
  "ListenFrom": [
    {
      "Type": "prometheus",
      "ListenAddr": "127.0.0.1:12003",
      "ListenPath": "/write"
    }
  ],
  "ForwardTo": [
    {
      "Type": "signalfx",
      "Name": "testgateway",
      "AuthTokenEnvVar": "SFX_GATEWAY_TOKEN",
      "URL": "https://ingest.signalfx.com/v2/datapoint"
    }
  ]
}
```

Is there a way to replicate this config with the splunk-otel-collector? I tried looking for a prometheus remote write receiver but could not find this as an option

**Comment by atoulme** [comment]
While the gateway is deprecated, it is still supported by our team. I recommend you keep using this solution for now as the OpenTelemetry project is developing a prometheus remote write receiver that we will make available for this type of use cases. Please feel free to contribute to its development here: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14751
